Type 'ReactNode' is not assignable to type 'ReactElement'

We are still seeing the bug reported by #100 despite the fix being present in the plugin release we deployed (2.4.0)

Our implementation is a PoC and we followed the exact same steps as documented in the ldap plugin readme, no customizations for the frontend, only the login form and a button (default).

Authentication works when backstage is executed locally but when we try to generate a docker image and execute `yarn tsc` we receive the following error:

```
packages/app/src/App.tsx:66:8 - error TS2322: Type '{ provider: string; onSignInSuccess(identityApi: IdentityApi): void; children?: ReactNode; }' is not assignable to type '{ provider: string; children?: ReactElement<any, string | JSXElementConstructor<any>> | null | undefined; options?: { helperTextPassword?: string | undefined; helperTextUsername?: string | undefined; validateUsername?: ((usr: string) => boolean) | undefined; validatePassword?: ((pass: string) => boolean) | undefined...'.
  Types of property 'children' are incompatible.
    Type 'ReactNode' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>> | null | undefined'.
      Type 'string' is not assignable to type 'ReactElement<any, string | JSXElementConstructor<any>>'.

66       <LdapAuthFrontendPage {...props} provider="ldap" />
```

We are not TypeScript/React developers but we succeeded running `yarn tsc` after replacing the following line of code under `node_modules/@immobiliarelabs/backstage-plugin-ldap-auth/dist/index.d.ts` :

```tsx
declare type LdapSignInPageProps = SignInPageProps & {
    ...
    children?: React.ReactNode | null;
    ...
};
```

Don't know if the fix is valid from the plugin perspective reason why we do not propose it through a PR.

--- a/packages/ldap-auth/src/components/LoginPage/LoginPage.tsx
+++ b/packages/ldap-auth/src/components/LoginPage/LoginPage.tsx
@@ -32,7 +32,7 @@ import { LoginForm } from './Form';
  */
 export type LdapSignInPageProps = SignInPageProps & {
     provider: string;
-    children?: React.ReactElement | null;
+    children?: React.ReactNode | null;
     options?: {
         helperTextPassword?: string;
         helperTextUsername?: string;

